### PR TITLE
Use cookie policy package

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Replaced the cookie policy code with the cookie-policy npm package.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check that the cookie policy is shown (you might need to use a private browser window).

### CODE REVIEW

File: static/sass/styles.scss [hunk 1]
```diff
@@ -7,6 +7,9 @@
 // custom mixins
 @import "mixins/full-width-breakout";
 
+// import cookie policy
+@import "cookie-policy/build/css/cookie-policy";
```
Comment (anthonydillon) [inline static/sass/styles.scss]: I don't think you need these. I believe the js brings in the CSS.

**Comment by anthonydillon** [comment]
Can you bump the timer from 3 seconds to something more.

File: static/sass/styles.scss (hunk 1 above)
Comment (huwshimi) [reply static/sass/styles.scss]: I just double checked the JS, but without the css include it appeared unstyled.

**Comment by huwshimi** [comment]
@anthonydillon I've updated it to appear for 5 seconds (same as blog.ubuntu.com). Sound OK?

**Comment by huwshimi** [comment]
@anthonydillon I've updated the timeout to be 10 seconds, to match the current timeout on ubuntu.com.

File: .gitignore [hunk 2]
```diff
@@ -60,5 +60,10 @@ env[23]/
 # Project-specific ignores
 etc/ubuntu-mirrors-rss.xml
 .search-api-key
+<<<<<<< HEAD
```
Comment (anthonydillon) [inline .gitignore]: Cornflakes 🤪

**Pull request merged**